modules/uhttp/filters.go

```diff
@@ -68,11 +69,16 @@ func tracingServerFilter(host service.Host) FilterFunc {
 		if err != nil && err != opentracing.ErrSpanContextNotFound {
 			fxctx.Logger().Info("Malformed inbound tracing context: ", "error", err.Error())
 		}
-
 		span := opentracing.GlobalTracer().StartSpan(operationName, ext.RPCServerOption(spanCtx))
 		ext.HTTPUrl.Set(span, r.URL.String())
 		defer span.Finish()
 
+		jSpanCtx, ok := span.Context().(jaeger.SpanContext)
+		if ok {
+			fxctx.Logger().Info(
```

Comment:
Check description. This is not directly related to the task. This is for debugging.